### ISSUE TITLE
How can I cancel calendar item in 3.2.0?

### ISSUE THREAD

**Issue by LiChaozhong** [ecederstrand/exchangelib#791]
from  [755](https://github.com/ecederstrand/exchangelib/issues/755)

I need to use CalendarItem.occurrence(), so I have to use 3.2.0 rather than 3.1.1

**Comment by ecederstrand** [comment]
CalendarItem objects have a `.cancel()` method for this purpose.

**Comment by LiChaozhong** [comment]
> CalendarItem objects have a `.cancel()` method for this purpose.

but it can't work

```
File "/Users/lichaozhong01/Documents/baidu/soqa/workflow/testing/interfaceTest/common/utils/ews_helper.py", line 70, in ews_cancel
    calendar_item.cancel()
  File "/usr/local/lib/python3.7/site-packages/exchangelib/items/calendar_item.py", line 192, in cancel
    **kwargs
  File "/usr/local/lib/python3.7/site-packages/exchangelib/util.py", line 39, in wrapper
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/exchangelib/items/base.py", line 190, in send
    expect_result=False,
  File "/usr/local/lib/python3.7/site-packages/exchangelib/services/common.py", line 72, in get
    raise ValueError('Expected result length 0, but got %r', res)
Exception: ('Expected result length 0, but got %r', [<Element {http://schemas.microsoft.com/exchange/services/2006/types}CalendarItem at 0x1064d59b0>])
```

**Comment by ecederstrand** [comment]
https://github.com/ecederstrand/exchangelib/commit/9268db066bfee802cb77bc7df8ba1edd8e4ee589 is still only on the master branch, so you'll have to install from that until the next release.

**Comment by ecederstrand** [comment]
I just released version 3.2.1 containing the fix.
